Aircompany/Aircompany/Planes/Plane.cs

```diff
@@ -4,68 +4,30 @@ namespace Aircompany.Planes
 {
     public abstract class Plane
     {
-        public string _model;
-        public int _maxSpeed;
-        public int _maxFlightDistance;
-        public int _maxLoadCapacity;
-
-        public Plane(string model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity)
```

Comment:
Return all the constructors, no need to avoid them.
If you want to use your approach, just create additional 0-params constructor